<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiderClass</class>
 <widget class="QWidget" name="guiderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Guiding</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_Drift">
         <property name="title">
          <string>Drift graphics</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_Graph">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="l_1">
              <property name="text">
               <string>X scale(frm.)</string>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_XScale">
              <property name="minimum">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="l_2">
              <property name="text">
               <string>Y scale(&quot;)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_YScale">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_Info">
         <property name="title">
          <string>Info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="l_3">
              <property name="text">
               <string>Guiding rate,x15&quot;/sec</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinBox_GuideRate">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="l_RecommendedGain">
              <property name="text">
               <string>P=xxxxx</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="l_5">
              <property name="text">
               <string>Focal,mm</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="l_Focal">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>xxx</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLabel" name="l_6">
              <property name="text">
               <string>Aperture,mm</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="l_Aperture">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>xx</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="l_7">
              <property name="text">
               <string>F/D</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="l_FbyD">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>xxx</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="l_8">
              <property name="text">
               <string>FOV,'</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3" colspan="2">
             <widget class="QLabel" name="l_FOV">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>YYxYY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="l_16">
              <property name="text">
               <string>Delta ,&quot;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="l_DeltaRA">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxxx</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="l_DeltaDEC">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxxx</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="l_17">
              <property name="text">
               <string>Pulse duration, ms</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="l_PulseRA">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxxx</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="l_PulseDEC">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxxx</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="l_20">
                <property name="text">
                 <string>Sig(RA)&quot;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="l_ErrRA">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxx</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="l_21">
                <property name="text">
                 <string>Sig(DEC)&quot;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="l_ErrDEC">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>xxx</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Configuration">
         <property name="title">
          <string>Setup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="captureB">
                <property name="text">
                 <string>Capture</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pushButton_StartStop">
                <property name="text">
                 <string>Start Autoguide</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="l_18">
                <property name="text">
                 <string>Square size</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_SquareSize"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="l_19">
                <property name="text">
                 <string>Algorithm</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="comboBox_ThresholdAlg"/>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="kfcg_guideSubFrame">
                <property name="toolTip">
                 <string>Subframe the image around the guide star</string>
                </property>
                <property name="text">
                 <string>Subframe</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="kfcg_useRapidGuide">
                <property name="text">
                 <string>Use Rapid Guide</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>32</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Control">
       <property name="title">
        <string>Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="l_9">
            <property name="text">
             <string>Enable directions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_DirRA">
            <property name="text">
             <string>RA</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_DirDEC">
            <property name="text">
             <string>DEC</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="l_10">
            <property name="text">
             <string>Accumulate frames</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_AccFramesRA">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_AccFramesDEC">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="l_11">
            <property name="text">
             <string>Proportional gain</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBox_PropGainRA">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_PropGainDEC">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="l_12">
            <property name="text">
             <string>Integral gain</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBox_IntGainRA">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_IntGainDEC">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="l_13">
            <property name="text">
             <string>Derivative gain</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBox_DerGainRA">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_DerGainDEC">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="l_14">
            <property name="text">
             <string>Maximum pulse</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBox_MaxPulseRA">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spinBox_MaxPulseDEC">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="l_15">
            <property name="text">
             <string>Minimum pulse</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_MinPulseRA">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spinBox_MinPulseDEC">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>111</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_XScale</tabstop>
  <tabstop>spinBox_YScale</tabstop>
  <tabstop>comboBox_SquareSize</tabstop>
  <tabstop>comboBox_ThresholdAlg</tabstop>
  <tabstop>spinBox_GuideRate</tabstop>
  <tabstop>checkBox_DirRA</tabstop>
  <tabstop>checkBox_DirDEC</tabstop>
  <tabstop>spinBox_AccFramesRA</tabstop>
  <tabstop>spinBox_AccFramesDEC</tabstop>
  <tabstop>spinBox_PropGainRA</tabstop>
  <tabstop>spinBox_PropGainDEC</tabstop>
  <tabstop>spinBox_IntGainRA</tabstop>
  <tabstop>spinBox_IntGainDEC</tabstop>
  <tabstop>spinBox_DerGainRA</tabstop>
  <tabstop>spinBox_DerGainDEC</tabstop>
  <tabstop>spinBox_MaxPulseRA</tabstop>
  <tabstop>spinBox_MaxPulseDEC</tabstop>
  <tabstop>spinBox_MinPulseRA</tabstop>
  <tabstop>spinBox_MinPulseDEC</tabstop>
  <tabstop>pushButton_StartStop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
